### PR TITLE
Scrutinizer and Travis-CI

### PR DESCRIPTION
**Pull request by fabiorphp**
Scrutinizer and Travis-CI configuration
Update Readme info

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,28 +1,28 @@
 language: php
 
+php:
+   - 5.4
```
Comment (naxhh) [inline .travis.yml]: Why are you reverting changes here?

File: .travis.yml [hunk 2]
```diff
@@ -1,28 +1,28 @@
...
+
 matrix:
-  include:
-    - php: 5.5
-    - php: 5.6
-      env: SCRUTINIZER_REPORT=true
-    - php: 7.0
   allow_failures:
     - php: 7.0
+    - php: hhvm
```
Comment (naxhh) [inline .travis.yml]: We are not expecting to run in hhvm so I think this is not useful

Comment (InFog) [inline .travis.yml]: Yep, not useful at all. Let's focus on our environment.

Comment (fabiorphp) [inline .travis.yml]: OK! OK!

**Comment by fabiorphp** [comment]
Done!

**Comment by naxhh** [comment]
I think it's fine, someone can also check it?

**Comment by iambrosi** [comment]
:+1:

**Pull request merged**